Source/Core/Matrix4.js

```diff
@@ -2328,7 +2330,8 @@ Matrix4.getTranslation = function (matrix, result) {
 };
 
 /**
- * Gets the upper left 3x3 rotation matrix of the provided matrix, assuming the matrix is an affine transformation matrix.
+ * Gets the upper left 3x3 rotation matrix of the provided matrix,
+ * assuming the matrix is a proper rigid transformation matrix.
```

Comment:
I understand that the function returns the upper left 3x3, but if it intends to return a "rotation" matrix, shouldn't we be more precise with the language? I would think that if this were intended to be used more generally, it would have just said "Get the upper left 3x3 matrix of the provided matrix." without further detail

Reply:
The original wording does seem a bit odd and inconsistent. It mentions rotation matrix, but the example it gives is the more general case.

>"Get the upper left 3x3 matrix of the provided matrix."

I like this as a description. It's simple and is consistent with the example